.github/workflows/asset-size-check.yml

```diff
@@ -0,0 +1,47 @@
+name: AssetSizeCheck
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  asset-size-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Checkout master
```

Comment:
Originally this is what we were doing (and then checking it in), after discussion we decided we didn't like storing derived info on a per-commit basis, nor the extra commits that brings.

Right now I'm moving to the dual build strategy but we should set up a way to get built artifacts for master to avoid the extra builds in the future.

Reply:
After further investigation, we can't use github's artifacts feature for this. At a future time we should explore an S3 or similar solution.